adserver/templates/adserver/reports/advertiser.html

```diff
@@ -44,6 +44,11 @@ <h2 class="col-md-8">{% trans 'Total results for all ads across all flights' %}<
 
         {% if export_url %}
           <aside class="mb-3 col-md-4 text-right">
+            <a href="{% url 'advertiser_per_ad_report' advertiser.slug %}" class="btn btn-sm btn-outline-secondary" role="button" aria-pressed="true">
```

Comment:
I wonder about putting it in the sidebar under `Reports` as well? Maybe after we test it a bit more?

Reply:
Sounds good.